packages/block-editor/src/components/block-list/block.js

```diff
@@ -161,7 +162,7 @@ function BlockListBlock( {
 			'is-focused':
 				isFocusMode && ( isSelected || isAncestorOfSelectedBlock ),
 			'is-focus-mode': isFocusMode,
-			// Don't select child while dragging.
+			// Same as "is-selected" above, we don't want the parent menu to be opened while dragging.
```

Comment:
The block component is a component that's used for all blocks, so I don't think the references to menus and submenus are clear enough for any other person reading.

Personally I think it's fine to remove these comments.

Reply:
Makes sense! I'll remove the comments.